docs/manual/java/gettingstarted/index.toc

```diff
@@ -2,10 +2,8 @@ IntroGetStarted:Introduction
 JavaPrereqs:Lagom Java prerequisites;next=GettingStartedMaven,GettingStartedSbt
 GettingStartedMaven:Creating and running Hello World with Maven;next=UnderstandHello
 GettingStartedSbt:Creating and running Hello World with sbt;next=UnderstandHello
-UnderstandHello:Understanding Hello World;next=MavenIDEs,EclipseSbt,IntellijSbt
-MavenIDEs:Integrating Maven projects with an IDE;next=LagomExamples
-EclipseSbt:Integrating sbt with Eclipse;next=LagomExamples
-IntellijSbt:Integrating sbt with IntelliJ;next=LagomExamples
+UnderstandHello:Understanding Hello World
 LagomExamples:Lagom examples
+WhatsNext:What's Next?
```

Comment:
I've been using capitalizing only the first letter of the first word quite consistently, but I never realised it was localisation issue, I just chose that because when I read headings I find it very unnatural to have each word capitalized, I end up putting an emphasis on every capitalised word when I read it in my head, akin to the way when you read all caps you tend to read it as shouting. But maybe it's a cultural influence that leads me to read it that way, I don't know.